data/mods/Magiclysm/recipes/weapons.json

```diff
@@ -0,0 +1,234 @@
+[
+  {
+    "type": "recipe",
+    "result": "rune_magus_weapon",
+    "\\": "Forging and water included for glass heat-tempering process",
+    "category": "CC_ENCHANTED",
+    "subcategory": "CSC_ENCHANTED_WEAPONS",
+    "skill_used": "fabrication",
+    "skills_required": [ [ "spellcraft", 2 ], [ "bashing", 2 ] ],
+    "difficulty": 4,
+    "time": "360 m",
+    "autolearn": true,
+    "using": [ [ "forging_standard", 2 ] ],
+    "qualities": [ { "id": "CHISEL", "level": 1 }, { "id": "MANA_INFUSE", "level": 1 } ],
+    "tools": [ [ [ "water", -1 ] ], [ [ "tongs", -1 ] ] ],
+    "components": [
+      [ [ "q_staff", 1 ] ],
+      [ [ "jar_glass", 2 ] ],
+      [ [ "rune_magus", 2 ] ],
+      [ [ "tallow", 2 ], [ "tallow_tainted", 2 ], [ "cooking_oil", 16 ], [ "cooking_oil2", 16 ], [ "lamp_oil", 250 ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "result": "rune_biomancer_weapon",
+    "category": "CC_ENCHANTED",
+    "subcategory": "CSC_ENCHANTED_WEAPONS",
+    "skill_used": "fabrication",
+    "skills_required": [ [ "spellcraft", 2 ], [ "piercing", 2 ] ],
+    "difficulty": 4,
+    "time": "2 m",
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 1 } ],
+    "tools": [ [ [ "fire", -1 ] ] ],
+    "components": [
+      [ [ "stick_long", 1 ], [ "pool_cue", 1 ] ],
+      [ [ "rag", 1 ], [ "felt_patch", 1 ], [ "leather", 1 ], [ "fur", 1 ] ],
+      [ [ "duct_tape", 20 ], [ "filament", 20, "LIST" ] ],
+      [ [ "bone", 3 ], [ "bone_human", 3 ], [ "bone_tainted", 6 ] ],
+      [ [ "rune_biomancer", 1 ] ]
+    ]
+  },
+  {
+    "type": "recipe",
+    "result": "rune_animist_weapon",
+    "category": "CC_ENCHANTED",
+    "subcategory": "CSC_ENCHANTED_WEAPONS",
+    "skill_used": "fabrication",
+    "skills_required": [ [ "spellcraft", 2 ], [ "piercing", 2 ] ],
+    "difficulty": 4,
+    "time": "360 m",
+    "autolearn": true,
+    "using": [ [ "forging_standard", 2 ], [ "steel_tiny", 1 ] ],
+    "qualities": [
+      { "id": "ANVIL", "level": 3 },
+      { "id": "HAMMER", "level": 3 },
+      { "id": "CHISEL", "level": 3 },
+      { "id": "MANA_INFUSE", "level": 1 }
+    ],
+    "tools": [ [ [ "tongs", -1 ] ], [ [ "swage", -1 ] ] ],
+    "components": [ [ [ "2x4", 1 ], [ "stick", 1 ] ], [ [ "rune_animist", 1 ] ] ]
+  },
+  {
+    "type": "recipe",
+    "result": "rune_druid_weapon",
+    "category": "CC_ENCHANTED",
+    "subcategory": "CSC_ENCHANTED_WEAPONS",
+    "skill_used": "fabrication",
+    "skills_required": [ [ "spellcraft", 2 ], [ "archery", 2 ] ],
+    "difficulty": 4,
+    "time": "360 m",
+    "autolearn": true,
+    "qualities": [ { "id": "CUT", "level": 1 }, { "id": "MANA_INFUSE", "level": 1 } ],
+    "components": [
+      [ [ "stick", 3 ], [ "2x4", 2 ] ],
```

Comment:
this recipe seems very cheap at first glance. how does this compare to the actual recipes for other bows? an acceptable alternative is to just infuse a bow with the druid rune.

Reply:
It's identical ingredients to the composite bow + 2 runes.

Compared to recurve bow(easiest crafted bow with more than dampening mod slots):
1) Fab 4/Spell2/Archery 2 vs. Fab 5/Mech 2
2) 9 less damage, 1 less ranged, 12.5% more dispersion compared to recurve
3) Recurve requires: fine metal sawing, hammering 2, one chunk of metal, duct tape/glue, and plastic chunks

Reply:
I'll just change it to composite bow infusion.

Reply:
that's probably more future-proofed, since i know erk was talking about changing how bow crafting works some time.